invesalius/gui/task_navigator.py

```diff
@@ -370,7 +362,7 @@ def __init__(self, parent):
         btn_nav.Bind(wx.EVT_TOGGLEBUTTON, partial(self.OnNavigate, btn=(btn_nav, choice_trck, choice_ref)))
 
         # Image and tracker coordinates number controls
-        for m in range(0, 7):
+        for m in range(0, len(self.btns_coord)):
```

Comment:
The 0 (zero) in the range is not needed, just do:
`for m in range(len(self.btns_coord)):`